str/str_iterative_eh_runner.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+"""
+This script uses ExpansionHunterv5 to call STRs on WGS cram files.
+Required input: --ehregions (file path to variant catalog) and external sample IDs 
+For example: 
+analysis-runner --access-level test --dataset tob-wgs --description "tester" --output-dir "tester" str_iterative_eh_runner.py --ehregions=gs://cpg-tob-wgs-test/hoptan-str/Illuminavariant_catalog.json TOB1XXXX TOB1XXXX
+
+Required packages: sample-metadata, hail, click, os 
+pip install sample-metadata hail click
+
+"""
+import os
+import hailtop.batch as hb
+import click
+
+from sample_metadata.model.analysis_type import AnalysisType
+from sample_metadata.model.analysis_query_model import AnalysisQueryModel
+from sample_metadata.apis import AnalysisApi, SampleApi
+from sample_metadata.models import AnalysisStatus
+from cpg_utils.config import get_config
+
+from cpg_utils.hail_batch import (
+    authenticate_cloud_credentials_in_job,
+    copy_common_env,
+    remote_tmpdir,
+    output_path
+)
+
+config = get_config()
+
+DATASET = config['workflow']['dataset']
+OUTPUT_SUFFIX = config['workflow']['output_prefix']
+BILLING_PROJECT = config['hail']['billing_project']
+REF_FASTA = os.path.join(config['workflow']['reference_prefix'], 'hg38/v0/Homo_sapiens_assembly38.fasta')
+SAMTOOLS_IMAGE = os.path.join(config['workflow']['image_registry_prefix'], 'samtools:v0')
+EH_IMAGE = os.path.join(config['workflow']['image_registry_prefix'], 'expansionhunter:5.0.0')
+
+#inputs: 
+#variant catalog
+@click.option('--ehregions', help='Full path to Illumina Variants catalog')\
```

Comment:
Nitpick: should the input name be `--variant-catalog` to match the comment above?